f4tt5ylp.wp4.txt

```diff
@@ -0,0 +1 @@
+Merge of espruino/master into efm32integration
```

Comment:
Good spot - looks like it's a temporary file created by Git (I think so you can edit the commit message). Not sure how it happened - maybe if you do a `git commit` in one window and edit the message, then do it again in a second window.